dom.bs

```diff
@@ -7115,6 +7117,11 @@ string <var>namespace</var> (default null):</p>
 <a for=/>attribute</a> <var>attr</var> and an <a for=/>element</a> <var>element</var>:
 
 <ol>
+ <li><p>Let <var>verifiedValue</var> be the result of calling <a abstract-op>get
+ Trusted Types-compliant attribute value</a> with <var>attr</var>'s <a for=Attr>local name</a>,
```

Comment:
I don't understand why this is an abstract-op. Generally outside of TC39 we should not be using that.

The wrapping is also wrong for DOM (applies several times). There's no wrapping inside of phrasing-level elements to make searching easier.